.gitignore

```diff
@@ -1 +1,2 @@
 /coverage.out
+/.idea
```

Comment:
Hi, Eugene, thank you for your 1st contribution in this project.

It would be better to not keep in project .gitignore all of developer tools files, like IDEs, because we all have different  preferred IDEs and tools.

Could you, please, ignore this dir by your global .gitignore, it could be made by
putting `.idea` in file ~/.gitignore and executing next cmd: `git config --global core.excludesfile '~/.gitignore'`.

Reply:
Fixed

Reply:
Thank you.